logging/logging_test.go

```diff
@@ -327,28 +327,56 @@ func TestToLogEntry(t *testing.T) {
 				HTTPRequest: &logging.HTTPRequest{
 					Request: &http.Request{
 						URL:    u,
-						Header: http.Header{"X-Cloud-Trace-Context": {"105445aa7843bc8bf206b120001000/000000000000004a;o=0"}},
+						Header: http.Header{"X-Cloud-Trace-Context": {"105445aa7843bc8bf206b12000100000/0000000000000001;o=0"}},
 					},
 				},
 			},
 			want: &logpb.LogEntry{
-				Trace:        "projects/P/traces/105445aa7843bc8bf206b120001000",
-				SpanId:       "000000000000004a",
+				Trace:        "projects/P/traces/105445aa7843bc8bf206b12000100000",
+				SpanId:       "0000000000000001",
+				TraceSampled: true,
+			},
+		}, {
+			name: "X-Trace-Context header with all fields; Span ID properly encoded in hexadecimal",
+			in: logging.Entry{
+				HTTPRequest: &logging.HTTPRequest{
+					Request: &http.Request{
+						URL:    u,
+						Header: http.Header{"X-Cloud-Trace-Context": {"105445aa7843bc8bf206b12000100000/16;o=1"}},
+					},
+				},
+			},
+			want: &logpb.LogEntry{
+				Trace:        "projects/P/traces/105445aa7843bc8bf206b12000100000",
+				SpanId:       "0000000000000010",
 				TraceSampled: true,
 			},
+		}, {
+			name: "X-Trace-Context with Span ID too large",
```

Comment:
Is there any logging or other debug information that can be output in such a case?